cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AuthenticationUtils.java

```diff
@@ -147,25 +229,56 @@ public static void configureClientAuthenticationVolumeMounts(KafkaClientAuthenti
      * @param envVarNamer   Function for naming the environment variables (ech components is using different names)
      */
     public static void configureClientAuthenticationEnvVars(KafkaClientAuthentication authentication, List<EnvVar> varList, Function<String, String> envVarNamer)   {
+        if (authentication != null) {
+
+            getClientAuthenticationProperties(authentication).entrySet().stream()
+                .forEach(entry -> varList.add(AbstractModel.buildEnvVar(envVarNamer.apply(entry.getKey()), entry.getValue())));
```

Comment:
Unless you're just consuming an existing stream there's no need to use `forEach`, just for a `for` statement.

Reply:
Done